packages/react/src/utils/format-date.tsx

```diff
@@ -0,0 +1,6 @@
+const formatDate = (stringDate: string | Date) => {
```

Comment:
Try to replace https://js.devexpress.com/Documentation/Guide/Common/Value_Formatting/?search=format